app/controllers/judgement_controller.rb

```diff
@@ -0,0 +1,33 @@
+class JudgementController < ApplicationController
+  before_action :require_login
+  before_action :require_privileges_of_rvt
+
+  def index
+    semester = SystemAttribute.semester.to_s
+    @evaluations = Evaluation.where(date: semester).page(params[:page]).decorate
+  end
+
+
+  def edit
+    @evaluation = Evaluation.find(params[:evaluation_id])
+  end
+
+  def update
+    unless SystemAttribute.evaluation_season?
+      return redirect_back fallback_location: judgements_path, alert: t(:not_evaluation_season)
+    end
+    evaluation = Evaluation.find(params[:evaluation_id])
+    success = CreateJudgement.call(evaluation, judgement_params)
```

Comment:
Is it necessary to store the return value in a variable? You could just use the function in the condition.